transport/internal/safelycall.go

```diff
@@ -56,3 +56,21 @@ func SafelyCallHandler(h transport.Handler, start time.Time, ctx context.Context
 
 	return err
 }
+
+// SafelyCallRegistry calls procedures on the given registry safely.
+//
+// See also SafelyCallHandler.
+func SafelyCallRegistry(
```

Comment:
As discussed offline - let's remove this function and just get the handler procedurally where needed.